package.json

```diff
@@ -2,11 +2,44 @@
   "name": "heimdalfrontend",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
+  "main": "index.jsx",
   "scripts": {
+    "build": "webpack --config webpack.prod.js",
```

Comment:
Hello @edogbosunny, nice work so far. Please change this build to start in order for Heroku to start the app